### ISSUE TITLE
#-1397039196------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1848]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<div id="attr-label-visibility" class="sc-dUWDJJ eBsayS" aria-describedby="calcite-tooltip-d7aebeed-3796-92af-7ba0-7ba819ac8138">visibility</div>`

<strong>Element Location: </strong>#attr-label-visibility

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-1397039196_All page content should be contained by landmarks.png`
